elements/map/package.json

```diff
@@ -7,10 +7,12 @@
     "flatgeobuf": "^3.35.0",
     "lit": "^3.0.2",
     "ol": "^10.2.0",
+    "ol-ext": "^4.0.24",
```

Comment:
Yes, this would be the first time using it, but probably not the last time. You're right, it could cause troubles when updating :thinking:
An alternative would be implementing this functionality ourselves, where I would hope for support from @RobertOrthofer

Reply:
It does look like a bit of an overkill for this specific feature, as  implementation would be quite trivial, something along the way of this:
```JS
  map.on('pointermove', (e) => {
    if (e.dragging) return;
    map.getTargetElement().style.cursor = map.hasFeatureAtPixel(pixel, {layerfilter, ...options}) ? 'pointer' : ''
  }
```
but I'm not strictly against using `ol-ext`, they offer some useful or fancy stuff, and implementations are quite clean for the most part.

... there might be an argument for not having this as an interaction that is added with a select layer, but rather have the pointer-changer as something the map simply does. The information if the pointer should change should be available on each layer, so if there is a feature at the pixel, and the layer of the feature has an active select interaction, the cursor could be changed to `pointer`.

The pro of this is the possibility of multiple hover-interactions without interfering, the con is that I don't see an easy way of doing this with a `hitTolerance`

Reply:
Difficult decision, am i understanding that in principle we are fine with this? So we go for it, and worst case if it creates trouble down the line we re-evaluate if we keep ol-ext? I would mark this as resolved then.

Reply:
Just to have another option, I have created an alternative in https://github.com/EOX-A/EOxElements/pull/1348; it uses the pointermove event as @RobertOrthofer suggested, which makes the entire thing much simpler. The only issue I am currently facing is that I didn't find a way to test this.